packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -224,9 +227,9 @@ export default function LabeledField(props: Props) {
             <React.Fragment>
                 <View
                     style={[
-                        styles.helperTextSection,
+                        styles.helperTextWithIcon,
```

Comment:
Rename styles for clarity:
- `helperTextSection` -> `helperTextWithIcon`
- `helperTextSectionWithContent` -> `spacingAboveHelperText`